document-portal/document-portal-fuse.c

```diff
@@ -2492,7 +2492,7 @@ xdp_fuse_readdir (fuse_req_t             req,
       if (off < d->dirbuf_size)
         {
           gsize reply_size = MIN (d->dirbuf_size - off, size);
-          g_autofree char *buf = g_memdup (d->dirbuf + off, reply_size);
+          g_autofree char *buf = g_memdup2 (d->dirbuf + off, reply_size);
```

Comment:
This is glib >= 2.68 but we only require >= 2.66

Reply:
The first commits bumps glib to 2.72

Reply:
nevermind I missed the bump in version in meson.build.